recipe/meta.yaml

```diff
@@ -47,7 +47,7 @@ requirements:
     # These are mostly run-like, but for the same reasons as
     # above, we need most of these at build time, so they are mostly
     # host-like
-    - boost =1.70
+    - boost =1.75
```

Comment:
Please do not do this. Nothing else in conda-forge is built against boost 1.75. This will make all environments in this package unsolvable at some level.

Reply:
Oops, sorry.  Where do I find an appropriate pin or range?

Reply:
I'd use any even version less than or equal to what is in the global pinnings: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L352